tests/OSPSuite.Presentation.Tests/Importer/Presenters/ImporterPresenterSpecs.cs

```diff
@@ -208,6 +208,81 @@ public void sets_column_mapping_presenter_settings()
       }
    }
 
+   public class When_setting_data_source_with_empty_rows : concern_for_ImporterPresenter
+   {
+      private RandomDataSourceFile _dataSourceFile;
+      protected Cache<string, DataSheet> _sheets;
+
+      protected override void Context()
+      {
+         base.Context();
+         _sheets = new Cache<string, DataSheet>();
+         _sheets.Add("Sheet1", A.Fake<DataSheet>());
+
+         _dataSourceFile = new RandomDataSourceFile(A.Fake<IImportLogger>());
+         _dataSourceFile.Format = A.Fake<IDataFormat>();
+         _dataSourceFile.Path = "FakePath";
+         A.CallTo(() => _importerDataPresenter.SetDataSource(A<string>.Ignored)).Returns(_dataSourceFile);
+         _importerDataPresenter.OnImportSheets += Raise.With(new ImportSheetsEventArgs()
+            { Filter = "", DataSourceFile = _dataSourceFile, SheetNames = _sheets.Keys.ToList() });
+      }
+
+      [Observation]
+      public void should_not_throw_an_ImporterParsingException()
+      {
+         sut.LoadConfiguration(_importerConfiguration, "");
+      }
+
+      //This class needs to be added to create a random data source file
+      //With the empty rows that we need to test
+      public class RandomDataSourceFile : DataSourceFile
```

Comment:
Could you use an existing integration test file? For example `IntegrationSampleUnitFromColumn.xlsx`. Just insert a row in the middle of the data?